src/python/ensembl/io/genomio/manifest_stats.py

```diff
@@ -17,70 +17,91 @@
 
 import gzip
 import io
-import json
 from pathlib import Path
 from shutil import which
 from statistics import mean
 import subprocess
 from typing import Dict, List, TextIO
-
+import argparse, os
 from BCBio import GFF
-import eHive
+import json
+
 
-from ensembl.brc4.runnable.utils import get_json
+def get_args():
+    """
+    Handle script arguments
+    :return: Script arguments
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-m', '--manifest_dir', help='Manifest file path', type=str, required=True)
+    parser.add_argument('-a', '--accession', help='Accession_ID', type=str, required=True)
+    parser.add_argument('-e', '--error', help='error status, True or False, Default False', default="false")
+    parser.add_argument('-dsin', '--datasets_bin', help='Datasets_bin Status', type=str, required=True)
```

Comment:
We'll probably want to change that to use ArgSchema for uniformity in the future